### PR TITLE
make test output less verbose

### PR DESCRIPTION
**Pull request by brittonsmith**
The log output from travis is so long as to make it difficult to find the errors. This sets the loglevel to warn to hopefully bring that down a bit.  I also uncommented the python 3 and minimum dependency python 2 tests because it looks like they get run anyway.

### CODE REVIEW

**Comment by ngoldbaum** [comment]
I think nose might still capture the logging even with this. Maybe you want to run `nosetests` with `—nologcapture` I’d add `-v` as well so nose prints which test it’s running instead of dots.

**Comment by brittonsmith** [comment]
Ok, good call. I'll do that and restart these.

**Comment by brittonsmith** [comment]
It's probably worth it to add h5py as a hard dependency in setup.py since most of the modules here need it.

**Comment by ngoldbaum** [comment]
Agreed

File: setup.py [hunk 1]
```diff
@@ -182,7 +183,7 @@ def run(self):
     },
     cmdclass={'sdist': sdist, 'build_ext': build_ext, 'build_py': build_py},
     author="The yt project",
-    author_email="yt-dev@lists.spacepope.org",
```
Comment (ngoldbaum) [inline setup.py]: good catch!

**Pull request merged**